backend-issuer/src/env.ts

```diff
@@ -11,14 +11,18 @@ export default envalid.cleanEnv(process.env, {
   PORT: envalid.port({ default: 3000 }),
   LOG_LEVEL: envalid.str({ default: 'info', devDefault: 'debug' }),
   CLOUDAGENT_HOST: envalid.host({ devDefault: 'localhost' }),
-  CLOUDAGENT_PORT: envalid.port({ default: 3000, devDefault: 3001 }),
-  DB_HOST: envalid.str({ devDefault: 'localhost' }),
+  CLOUDAGENT_PORT: envalid.port({ default: 3000, devDefault: 3010 }),
+  DB_HOST: envalid.str({ default: 'postgres', devDefault: 'localhost' }),
   DB_PORT: envalid.port({ default: 5432 }),
-  DB_USERNAME: envalid.str({ devDefault: 'postgres' }),
-  DB_PASSWORD: envalid.str({ devDefault: 'postgres' }),
+  DB_USERNAME: envalid.str({ default: 'postgres' }),
+  DB_PASSWORD: envalid.str({ default: 'postgres' }),
```

Comment:
Shouldn't these be devDefaults?  Thus requiring the prod env to always have them specified?

Reply:
agreed, changed them to devDefault